community/event-checklist.md

```diff
@@ -0,0 +1,95 @@
+toc=true
+
++++
+
+# 1. Logistics - Before the Workshop
+
+✔️ Decide the **_date and time_** of the event as per the convenience of speakers and audience. Conducting **_events during exams_** or some other important event might **_not be a good idea_** in general. Checking **_academic calendar_** and other events in KGP is suggested.
```

Comment:
Does markdown now also support emojis for list decoration or is it just a stylistic change? If so why not use the default unordered list decorators ex. `-` & `*`?

Reply:
The reason I used ✔️ for points was that I wanted the reader to feel them as a checklist rather than just some points. While conducting a workshop, keep this list in a dropbox paper or github issue and verify that all important things are checked.

It might seem trivial, but there are events in the past, where we have missed some of them. Hence I felt a check list was important.

Also, the checkmark emojis will be rendered on web too, when this doc will be a static site.